domain/src/main/java/org/fao/geonet/repository/UserSavedSelectionRepositoryCustomImpl.java

```diff
@@ -131,9 +131,11 @@ public int deleteAllBySelection(Integer selectionId) {
         final String selectionIdPath =
             SortUtils.createPath(UserSavedSelection_.id, UserSavedSelectionId_.selectionId);
         final String qlString =
-            "DELETE FROM " + UserSavedSelection.class.getSimpleName() +
-                " WHERE " + selectionIdPath + " = " + selectionId;
-        final int deleted = _entityManager.createQuery(qlString).executeUpdate();
+            "DELETE FROM UserSavedSelection WHERE :selectionIdPath = :selectionId";
+        final int deleted = _entityManager.createQuery(qlString)
+            .setParameter("selectionIdPath", selectionIdPath)
+            .setParameter("selectionId", selectionId)
+            .executeUpdate();
```

Comment:
You can't pass the name of a column as named parameter to a SQL query. Use this instead:
```suggestion
            String.format("DELETE FROM UserSavedSelection WHERE %s = :selectionId", selectionIdPath);
        final int deleted = _entityManager.createQuery(qlString)
            .setParameter("selectionId", selectionId)
            .executeUpdate();
```